src/main/scala/EnsimePlugin.scala

```diff
@@ -242,11 +242,15 @@ object EnsimePlugin extends AutoPlugin {
       }
     }
 
+  private val isPreJava9 = sys.props("java.version").contains(".")
```

Comment:
With Java `9.0.1` this would fail, so checking if the version string starts with `1` or comparing the major version against `9` should work.

Reply:
> With Java 9.0.1 this wouldn't work, so checking if the version string starts with 1 or comparing the major version against 9 should work.

Yes, but your logic will then fail with Java 10 again...

I changed the code to use `java.specifiction.version`, which should work correctly according to http://openjdk.java.net/jeps/223